app/views/report/_export_widgets.haml

```diff
@@ -73,7 +73,7 @@
 
     setTimeout(function() {
       if (typeof(dhxLayoutB) !== 'undefined') {
-        dhxLayoutB.cells('c').collapse();
+        ManageIQ.layout.toolbar.hide();
         $(dhxLayoutB.cells('c')).height(1);
```

Comment:
@skateman : Most likely waiting for some DOM node to exist. Surely not the best way to do things.